test/media-sources_test.js

```diff
@@ -1106,15 +1106,20 @@
     mediaSource.swfObj.vjs_endOfStream = function() {
```

Comment:
I think the description of this test should be updated now that we don't actually call `vjs_endOfStream` after the last append.